addons/sale/models/sale_config_settings.py

```diff
@@ -29,69 +19,26 @@ class SaleConfiguration(models.TransientModel):
-    group_discount_per_so_line = fields.Selection([
-        (0, 'No discount on sales order lines, global discount only'),
-        (1, 'Allow discounts on sales order lines')
-        ], "Discount",
-        implied_group='sale.group_discount_per_so_line')
-    group_display_incoterm = fields.Selection([
-        (0, 'No incoterm on reports'),
-        (1, 'Show incoterms on sales orders and invoices')
-        ], "Incoterms",
-        implied_group='sale.group_display_incoterm',
-        help="The printed reports will display the incoterms for the sales orders and the related invoices")
-    module_sale_margin = fields.Selection([
-        (0, 'Salespeople do not need to view margins when quoting'),
-        (1, 'Display margins on quotations and sales orders')
-        ], "Margins")
-    group_sale_layout = fields.Selection([
-        (0, 'Do not personalize sales orders and invoice reports'),
-        (1, 'Personalize the sales orders and invoice report with categories, subtotals and page-breaks')
-        ], "Sales Reports Layout", implied_group='sale.group_sale_layout')
-    group_warning_sale = fields.Selection([
-        (0, 'All the products and the customers can be used in sales orders'),
-        (1, 'An informative or blocking warning can be set on a product or a customer')
-        ], "Warning", implied_group='sale.group_warning_sale')
-    module_website_quote = fields.Selection([
-        (0, 'Print quotes or send by email'),
-        (1, 'Send quotations your customer can approve & pay online (advanced)')
-        ], "Online Quotations")
-    group_sale_delivery_address = fields.Selection([
-        (0, "Invoicing and shipping addresses are always the same (Example: services companies)"),
-        (1, 'Display 3 fields on sales orders: customer, invoice address, delivery address')
-        ], "Addresses", implied_group='sale.group_delivery_invoice_address')
+    group_uom = fields.Boolean("Units of Measure",
+        implied_group='product.group_uom')
+    group_discount_per_so_line = fields.Boolean("Discounts", implied_group='sale.group_discount_per_so_line')
+    group_stock_packaging = fields.Boolean("Packaging", implied_group='product.group_stock_packaging',
+        help="""Ability to select a package type in sales orders and 
+                to force a quantity that is a multiple of the number of units per package.""")
+    module_sale_margin = fields.Boolean("Margins")
+    group_sale_layout = fields.Boolean("Sales Reports Layout", implied_group='sale.group_sale_layout')
+    group_warning_sale = fields.Boolean("Warnings", implied_group='sale.group_warning_sale')
+    module_website_quote = fields.Boolean("Online Quotations & Templates")
+    group_sale_delivery_address = fields.Boolean("Customer Addresses", implied_group='sale.group_delivery_invoice_address')
+    fix_sale_price = fields.Boolean("A single sale price per product")
```

Comment:
And what do you do when ```sale_pricelist_setting = 'fixed'```

Could you please think during 2 seconds about what you do ?